RELEASE/scripts/autoscend/paths/license_to_adventure.ash

```diff
@@ -539,41 +538,17 @@ boolean LM_bond()
 			}
 		}
 
-		if((internalQuestStatus("questL08Trapper") < 2) && (my_level() >= 8))
+		if(internalQuestStatus("questL08Trapper") == 0 || internalQuestStatus("questL08Trapper") == 1)
 		{
-			if(item_amount($item[Goat Cheese]) == 0)
-			{
-				L8_trapperStart();
-				set_property("auto_combatDirective", "start;(olfaction)");
-				if((get_property("_kgbTranquilizerDartUses").to_int() < 3) && (item_amount($item[Kremlin\'s Greatest Briefcase]) > 0))
-				{
-					equip($slot[acc3], $item[Kremlin\'s Greatest Briefcase]);
-				}
-				if(L8_trapperGround())
-				{
-					set_property("auto_combatDirective", "");
-					return true;
-				}
-				set_property("auto_combatDirective", "");
-			}
-			else if(item_amount($item[Goat Cheese]) == 2)
+			/*
+			//TODO only run this if we can actually use timespinner to prevent infinite loop. after PR #610 is merged fix this and uncomment it
+			if(item_amount($item[Goat Cheese]) == 2 && timeSpinnerRemaining() >= 3)
```

Comment:
610 did get merged so should probably clean this up

Reply:
Thanks for the reminder. will do

Reply:
done.